index.html

```diff
@@ -7,6 +7,6 @@
   <title>Document</title>
 </head>
 <body>
-  
+  <div>Todo 컴포넌트</div>
```

Comment:
```suggestion
  <span>Todo 컴포넌트</span>
```

여기는 div보다 span이 좋은 것 같아요